index.html

```diff
@@ -307,8 +307,10 @@ <h2>JavaScript Property to Detect Preference</h2>
       <h2>GPC Support Resource</h2>
       <p>
         A site MAY produce a resource at a .well-known URL in order for a site to represent the fact
-        that it abides by GPC. The purpose of a GPC Support Resource is for a site to convey its
-        support for the Global Privacy Control. By default, an origin's support is <em>unknown</em>.
+        that the site is respecting the GPC signal for the user requesting the
+        resource. The purpose of a GPC Support Resource is for a site to convey
+        the site's support for the Global Privacy Control for the current
+        session. By default, an origin's support is <em>unknown</em>.
```

Comment:
What does "current session" mean?  This isn't a facetious question, but a real problem.  The notion of a session has never been well defined.

For background, when it comes to HTTP resources making claims (such as whether GPC is respected), they can only make claims about a single resource.  A well-known resource allows a resource to make statements for an origin.  That's a useful notion here, but if the goal is to narrow the scope of these claims based on other information (session, client jurisdiction, whatever), then the way in which that is narrowed needs to be carefully defined.  For one, the implications for HTTP caches start to look very interesting when you do that.

Answering the questions @j-br0 asks might be better way to start digging into this question of scope.